README.rst

```diff
@@ -58,7 +58,8 @@ Windows users can download the portable app from the
 GitHub `releases page <https://github.com/steko/totalopenstation/releases>`_.
```

Comment:
Change <https://github.com/steko/totalopenstation/releases> by <https://github.com/totalopenstation/totalopenstation/releases>

Reply:
I did not see this change ^^;